docs/S3_IAM_REST_API_&_CLI.md

```diff
@@ -0,0 +1,355 @@
+# Supported S3 IAM REST APIs and CLI Operations
+
+The S3 IAM REST APIs are a collection of resources and methods which are integrated with AWS Identity and Access Management (IAM) service. The IAM service lets an administrator to securely control access to the S3 resources. As an IAM administrator, you can control the authentication and authorizations for an ID to use API Gateway resources. We've listed the S3 IAM REST APIs and CLI operations that are supported by the CORTX-S3 Server Submodule. 
+
+<details>
+<summary>CreateAccount</summary>
+<p>
+
+The CreateAccount request lets you create an S3 IAM account. 
+
+| Request | Request body attributes  | Request Parameters    |
```

Comment:
some APIs have response section as well. I guess we add response section to all APIs

Reply:
I have mentioned the response below the tables, try expanding a few and you will see them @amitwac2608

Reply:
yes Saumya, this is what I mentioned  - few APIs have response section, not all. 

So I added comment that "should we add response section for all APIs ? "

Reply:
I think we definitely should. Who can help me with these?

Reply:
file://shgpiwfsw001.ad.seagate.com/Data/753287/Downloads/S3%20IAM%20REST%20API.pdf